test.js

```diff
@@ -111,22 +111,22 @@ test('cache option', t => {
 
 test('promise support', async t => {
 	let i = 0;
-	const memoized = m(async () => i++);
+	const memoized = m(() => Promise.resolve(i++));
```

Comment:
Why did you change this test file?

Reply:
The xo linter complained about having an async function without an await inside. It's specified as `"xo": "*"` in dependencies, so i guess there was some change of this rule in some newer version.